scenes/game_elements/props/tree/tree.tscn

```diff
@@ -8,6 +8,7 @@
 radius = 16.0312
 
 [node name="Tree" type="Node2D"]
+scale = Vector2(0.858137, 0.870245)
```

Comment:
This change is caused by a bug in the tree scene which I have since fixed: #867.

Please revert this change as follows:

```bash
git restore --source 8dc93665 scenes/game_elements/props/tree/tree.tscn
git add scenes/game_elements/props/tree/tree.tscn
git commit -m 'Revert tree scale change'
```